src/learn/learn.cpp

```diff
@@ -264,19 +319,25 @@ namespace Learner
 
         const double m = (1.0 - lambda) * t + lambda * p;
 
-        cross_entropy_eval =
+        Loss entropy{};
```

Comment:
Could you please rename the variable to `loss`?

Reply:
done

Reply:
Looks good to me.  Thank you.